build/import/HostAgnostic.props

```diff
@@ -35,7 +35,7 @@
     <PackageReference Include="Microsoft.VisualStudio.ProjectSystem.Query" />
     <PackageReference Include="Microsoft.VisualStudio.Validation" />
     <PackageReference Include="Microsoft.VisualStudio.Threading" />
-    <PackageReference Include="Microsoft.VisualStudio.ProjectSystem.Analyzers" />
+    <PackageReference Include="Microsoft.VisualStudio.ProjectSystem.Analyzers" PrivateAssets="all" />
```

Comment:
Might make sense to add a comment on why these are private. Same with the other analyzer package. Otherwise, that reason will be lost with time and in an internal email thread. :-P

Reply:
Done.